subsys/net/ip/Kconfig

```diff
@@ -509,6 +509,10 @@ config NET_MAX_CONTEXTS
 	  is used when listening or sending network traffic. This is very
 	  similar as one could call a network socket in some other systems.
 
+config ZVFS_OPEN_ADD_SIZE_NET
```

Comment:
Just wonder if we should do the same for the TLS sockets (for `config NET_SOCKETS_TLS_MAX_CONTEXTS`, if enabled). I think those TLS contexts should add up to the regular net contexts to represent the acutal limit.

Reply:
Sure, I can add sub-option for TLS contexts.